www/components/Notification.js

```diff
@@ -0,0 +1,29 @@
+import React from "react";
+import styles from "../styles/Notification.module.css";
+import classNames from "classnames";
```

Comment:
Do we need to install this (aka add to `package.json`).

Reply:
No